src/python/structurefinder/pyberny/__init__.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 NWChemEx-Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pluginplay as pp
+from simde import TotalEnergy
+
+
+class GeomoptViaPyberny(pp.ModuleBase):
+
+    def __init__(self):
+        pp.ModuleBase.__init__(self)
+        self.satisfies_property_type(TotalEnergy())
+        self.description("Performs PyBerny optimization")
+        self.add_input('molecule')
```

Comment:
The property type already provides you with the molecule, no need to add another one.
```suggestion
```